controllers/provisioners/eks/eks.go

```diff
@@ -31,6 +31,7 @@ const (
 	ProvisionerName                     = "eks"
 	defaultLaunchConfigurationRetention = 2
 	OverrideDefaultLabelsAnnotationKey  = "instancemgr.keikoproj.io/default-labels"
+	OsFamilyLabel				        = "instancemgr.keikoproj.io/os-family"
```

Comment:
WDYT regarding label vs. annotation.
Would it be confusing that some configurations are via annotations and others via labels?

Reply:
Yeah, i'm on the fence on this one as well. 

Arguments for an annotation:
- Aligns with how similar configurations are enabled. 

Arguments for a label:
- Selectable. We could, in the future, allow cluster operators to define 'defaults' in the ConfigMap based off of the family type (or any other selector). So I, as an operator, could define that for os-family=windows, use the windows AMI, windows SG etc - and namespace operators would just need to label their IG with the appropriate os-type.

Reply:
Do you need a selector for IGs? I would think you want to add this as a node label no?

Reply:
You do for the mentioned cluster operator use case, I think. 

Some way to group/filter IGs to apply default settings - i.e. windows node IGs get image: x, default node IGs get image y, bottlerocket node IGs get image z.

Reply:
Yeah, I guess we haven't decided / designed how the second part of AMI selection will work and whether it will use labelSelector to do that - the way the default /configmap feature is designed right now is not via updating / patching CRs, so if you are referring to the existing feature with the configmap the labelSelector won't help here.
I think we need to spend more time thinking how the AMI selection will look like

If we don't need it to be selectable, will annotation be more appropriate? if not, we can stick with a label.

Reply:
Yep, that sounds correct to me.

Reply:
I'll put it as an annotation for now, actually - to keep this independent of the conversation around ami selection / multiple defaults.